src/hdf5/osw_2ioda.py

```diff
@@ -146,7 +146,7 @@ def main(args):
     GlobalAttrs['datetimeReference'] = datetime.fromtimestamp(obs_data['dateTime'].min()).strftime("%Y-%m-%dT%H:%M:%SZ")
     # append platform to long description
     long_description = add_long_description(osw_source)
-    GlobalAttrs['platformLongDescription'].append(long_description)
+    GlobalAttrs['platformLongDescription'] + long_description
```

Comment:
this isn't correct should be `+=` to add to the string, tried with `+=` and got the expected output for global attribute 'platformLongDescription` 

pushing up that fix

Reply:
ah, oops! you're right, thanks for catching it.